src/solvers/NLPobjPerFlux.m

```diff
@@ -11,15 +11,13 @@
 % Prob          NLP problem structure
 %
 %OUTPUT
-% value         Objective flux / v.^2
+% value         Objective flux / sum(v.^2)^

```

Comment:
Ok, I updated the PR adding osense as a required field for the NLPproblem input of solveCobraNLP. That addresses definitions of objective functions but does not address the "warning" output, which should probably be an error.